### PR TITLE
Kops - Update to latest images for supported distros

### PR DESCRIPTION
**Pull request by hakman**
Mostly maintenance work.
/cc @rifelpet

### CODE REVIEW

File: config/jobs/kubernetes/kops/kops-periodics-distros.yaml [hunk 1]
```diff
@@ -203,11 +203,11 @@ periodics:
       - --env=KOPS_RUN_TOO_NEW_VERSION=1
       - --extract=release/stable
       - --ginkgo-parallel
-      - --kops-args=--image=ami-02eac2c0129f6376b
+      - --kops-args=
+      - --kops-image=679593333241/CentOS Linux 7 x86_64 HVM EBS ENA 2002_01-b7ee8a69-ee97-4a49-9e68-afaee216db2e-ami-0042af67f8e4dcc20.4
```
Comment (rifelpet) [inline config/jobs/kubernetes/kops/kops-periodics-distros.yaml:207]: hm any reason to use the `--kops-image` syntax here as opposed to `--kops-args=--image` everywhere else? its probably worth being consistent one way or another (i dont have a preference either way). and maybe remove the empty `--kops-args` above if we go with `--kops-image`.

Comment (hakman) [reply config/jobs/kubernetes/kops/kops-periodics-distros.yaml:206]: @justinsb tried using the args for this image but was not very successful. In the end he added a new field for the image. I updated all configs to use `--kops-image` now.

**Comment by rifelpet** [comment]
nice, i like that we can run the centos7 test anywhere now that we arent providing an AMI ID directly :)
/lgtm
/approve

**Comment by hakman** [comment]
Thanks @rifelpet. I think we should find a way to fix the grid next.

**Comment by rifelpet** [comment]
yea, i tried taking a look at the [docs](https://github.com/kubernetes/test-infra/blob/master/testgrid/config.md) the other day but couldnt figure out exactly what we need for all of our jobs

**Pull request merged**